### PR TITLE
[tests-only][full-ci] Removed scenarios that are only for ocis/reva

### PR DESCRIPTION
**Pull request by roshanlc**
## Description
This PR removes scenarios and feature files that are only for ocis/reva.

## Related Issue
Part of issue 
- https://github.com/owncloud/core/issues/40572

## How Has This Been Tested?

- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiAuth/cors.feature [hunk 1]
```diff
@@ -139,71 +139,3 @@ Feature: CORS headers
       | 2               | /cloud/groups | 200      | 200       |
       | 1               | /cloud/users  | 100      | 200       |
       | 2               | /cloud/users  | 200      | 200       |
-
-  @issue-34679 @skipOnOcV10
```
Comment (phil-davis) [inline tests/acceptance/features/apiAuth/cors.feature:143]: Issue #34679 is still open for oC10. There is `corsOc10Issue34679.feature` That demonstrates the current wrong behavior on oC10.

This scenario here has what we think is the expected behavior when the issue is fixed. Even though the scenario outline is skipped, we want it here for developers to use when they are fixing the issue.

File: tests/acceptance/features/apiAuth/cors.feature [hunk 2]
```diff
@@ -139,71 +139,3 @@ Feature: CORS headers
...
-      | 1               | /privatedata/getattribute                        | 997      | 401       |
-      | 2               | /privatedata/getattribute                        | 997      | 401       |
-      | 1               | /cloud/apps                                      | 997      | 401       |
-      | 2               | /cloud/apps                                      | 997      | 401       |
-      | 1               | /cloud/groups                                    | 997      | 401       |
-      | 2               | /cloud/groups                                    | 997      | 401       |
-      | 1               | /cloud/users                                     | 997      | 401       |
-      | 2               | /cloud/users                                     | 997      | 401       |
-
-    @files_external-app-required @notToImplementOnOCIS
```
Comment (phil-davis) [inline tests/acceptance/features/apiAuth/cors.feature:169]: But we can remove the `notToImplementOnOCIS` tag.

File: tests/acceptance/features/apiAuth/cors.feature [hunk 3]
```diff
@@ -139,71 +139,3 @@ Feature: CORS headers
...
-    Examples:
-      | ocs_api_version | endpoint                                         | ocs-code | http-code |
-      | 1               | /apps/files_sharing/api/v1/remote_shares         | 997      | 401       |
-      | 2               | /apps/files_sharing/api/v1/remote_shares         | 997      | 401       |
-      | 1               | /apps/files_sharing/api/v1/remote_shares/pending | 997      | 401       |
-      | 2               | /apps/files_sharing/api/v1/remote_shares/pending | 997      | 401       |
-      | 1               | /apps/files_sharing/api/v1/shares                | 997      | 401       |
-      | 2               | /apps/files_sharing/api/v1/shares                | 997      | 401       |
-
-  @issue-34679 @skipOnOcV10
```
Comment (phil-davis) [inline tests/acceptance/features/apiAuth/cors.feature:185]: Sane here - #34679 is still an open issue.

File: tests/acceptance/features/apiAuthOcs/ocsDELETEAuth.feature [hunk 4]
```diff
@@ -3,28 +3,3 @@ Feature: auth
 
   Background:
     Given user "another-admin" has been created with default attributes and without skeleton files
-
-  @smokeTest @issue-ocis-reva-30 @issue-ocis-reva-65 @skipOnBruteForceProtection @issue-brute_force_protection-112 @skipOnOcV10 @issue-32068
```
Comment (phil-davis) [inline tests/acceptance/features/apiAuthOcs/ocsDELETEAuth.feature:7]: Issue #32068 is still open. And there is `ocsDELETEAuthOc10Issue32068.feature` that demonstrates the problem.

File: tests/acceptance/features/apiAuthOcs/ocsGETAuth.feature [hunk 5]
```diff
@@ -4,27 +4,6 @@ Feature: auth
   Background:
     Given user "Alice" has been created with default attributes and without skeleton files
 
-  @issue-32068 @skipOnOcV10 @issue-ocis-reva-30 @smokeTest
```
Comment (phil-davis) [inline tests/acceptance/features/apiAuthOcs/ocsGETAuth.feature:7]: Issue #32068 is still open.

File: tests/acceptance/features/apiAuthOcs/ocsGETAuthFilesExternal.feature [hunk 6]
```diff
@@ -4,37 +4,6 @@ Feature: auth
   Background:
     Given user "Alice" has been created with default attributes and without skeleton files
 
-  @issue-32068 @skipOnOcV10 @smokeTest
```
Comment (phil-davis) [inline tests/acceptance/features/apiAuthOcs/ocsGETAuthFilesExternal.feature:7]: Issue #32068 is still open.

File: tests/acceptance/features/apiAuthOcs/ocsPUTAuth.feature [hunk 7]
```diff
@@ -19,13 +19,3 @@ Feature: auth
       | /ocs/v2.php/apps/files_sharing/api/v1/shares/123 |
     Then the HTTP status code of responses on all endpoints should be "401"
     And the OCS status code of responses on all endpoints should be "997"
-
-  @issue-38423 @skipOnOcV10
```
Comment (phil-davis) [inline tests/acceptance/features/apiAuthOcs/ocsPUTAuth.feature:23]: Issue #38423 is still open

**Review by phil-davis — COMMENTED** [review]
We need to leave the scenarios that are part of the oC10 "bug demonstration" system.

File: tests/acceptance/features/apiAuthWebDav/webDavCOPYAuth.feature [hunk 8]
```diff
@@ -161,24 +107,3 @@ Feature: COPY file/folder
...
-  Scenario: send COPY requests to webDav endpoints with body as normal user
-    When user "Alice" requests these endpoints with "COPY" including body "doesnotmatter" about user "Alice"
-      | endpoint                                           |
-      | /remote.php/webdav/textfile0.txt                   |
-      | /remote.php/dav/files/%username%/textfile0.txt     |
-      | /remote.php/webdav/PARENT                          |
-      | /remote.php/dav/files/%username%/PARENT            |
-      | /remote.php/webdav/PARENT/parent.txt               |
-      | /remote.php/dav/files/%username%/PARENT/parent.txt |
-    Then the HTTP status code of responses on all endpoints should be "415"
```
Comment (grgprarup) [inline tests/acceptance/features/apiAuthWebDav/webDavCOPYAuth.feature:175]: We have a bug demo for issue 40144. Not to remove.

File: tests/acceptance/features/apiAuthWebDav/webDavDELETEAuth.feature [hunk 9]
```diff
@@ -170,35 +105,3 @@ Feature: delete file/folder
...
-  @skipOnOcV10
-  Scenario: send DELETE requests to webDav endpoints with body as normal user
-    When user "Alice" requests these endpoints with "DELETE" including body "doesnotmatter" about user "Alice"
-      | endpoint                                           |
-      | /remote.php/webdav/textfile0.txt                   |
-      | /remote.php/dav/files/%username%/textfile1.txt     |
-      | /remote.php/dav/files/%username%/PARENT/parent.txt |
-      | /remote.php/webdav/PARENT                          |
-      | /remote.php/dav/files/%username%/FOLDER            |
-    Then the HTTP status code of responses on all endpoints should be "415"
```
Comment (grgprarup) [inline tests/acceptance/features/apiAuthWebDav/webDavDELETEAuth.feature:195]: We have a bug demo for issue 40144. Not to remove.

File: tests/acceptance/features/apiAuthWebDav/webDavMKCOLAuth.feature [hunk 10]
```diff
@@ -39,59 +30,6 @@ Feature: create folder using MKCOL
...
-  @skipOnOcV10 @issue-ocis-5049 @issue-ocis-reva-9 @issue-ocis-reva-197
-  Scenario: send MKCOL requests to non-existent user's webDav endpoints as normal user
-    Given user "Brian" has been created with default attributes and without skeleton files
-    When user "Brian" requests these endpoints with "MKCOL" including body "" about user "non-existent-user"
-      | endpoint                                                  |
-      | /remote.php/dav/files/non-existent-user/textfile0.txt     |
-      | /remote.php/dav/files/non-existent-user/PARENT            |
-      | /remote.php/dav/files/non-existent-user/does-not-exist    |
-      | /remote.php/dav/files/non-existent-user/PARENT/parent.txt |
-    Then the HTTP status code of responses on all endpoints should be "404"
```
Comment (grgprarup) [inline tests/acceptance/features/apiAuthWebDav/webDavMKCOLAuth.feature:71]: Need to verify whether these can be removed or not.

File: tests/acceptance/features/apiAuthWebDav/webDavMOVEAuth.feature [hunk 11]
```diff
@@ -157,24 +103,3 @@ Feature: MOVE file/folder
...
-  Scenario: send MOVE requests to webDav endpoints with body as normal user
-    When user "Alice" requests these endpoints with "MOVE" including body "doesnotmatter" about user "Alice"
-      | endpoint                                           |
-      | /remote.php/webdav/textfile0.txt                   |
-      | /remote.php/dav/files/%username%/textfile0.txt     |
-      | /remote.php/webdav/PARENT                          |
-      | /remote.php/dav/files/%username%/PARENT            |
-      | /remote.php/webdav/PARENT/parent.txt               |
-      | /remote.php/dav/files/%username%/PARENT/parent.txt |
-    Then the HTTP status code of responses on all endpoints should be "415"
```
Comment (grgprarup) [inline tests/acceptance/features/apiAuthWebDav/webDavMOVEAuth.feature:171]: We have a bug demo for issue 40144. Not to remove.

File: tests/acceptance/features/apiAuthWebDav/webDavMKCOLAuth.feature (hunk 10 above)
Comment (grgprarup) [inline tests/acceptance/features/apiAuthWebDav/webDavMKCOLAuth.feature:71]: We have a bug demo for issue 40485. Not to remove.

File: tests/acceptance/features/apiAuthWebDav/webDavPUTAuth.feature [hunk 12]
```diff
@@ -43,39 +34,6 @@ Feature: get file info using PUT
...
-  Scenario: send PUT requests to another user's webDav endpoints as normal user
-    When user "Brian" requests these endpoints with "PUT" including body "doesnotmatter" about user "Alice"
-      | endpoint                                       |
-      | /remote.php/dav/files/%username%/textfile1.txt |
-      | /remote.php/dav/files/%username%/PARENT        |
-    Then the HTTP status code of responses on all endpoints should be "403"
-    When user "Brian" requests these endpoints with "PUT" including body "doesnotmatter" about user "Alice"
-      | endpoint                                           |
-      | /remote.php/dav/files/%username%/PARENT/parent.txt |
-    Then the HTTP status code of responses on all endpoints should be "403"
```
Comment (grgprarup) [inline tests/acceptance/features/apiAuthWebDav/webDavPUTAuth.feature:65]: We have a bug demo for issue 39597. Not to remove.

File: tests/acceptance/features/apiMain/checksums.feature [hunk 13]
```diff
@@ -154,22 +154,6 @@ Feature: checksums
...
-    And user "Alice" has uploaded file "filesForUpload/textfile.txt" to "/myChecksumFile.txt" with checksum "MD5:d70b40f177b14b470d1756a3c12b963a"
-    And user "Alice" has shared file "/myChecksumFile.txt" with user "Brian"
-    And user "Brian" has accepted share "/myChecksumFile.txt" offered by user "Alice"
-    When user "Brian" uploads file with checksum "SHA1:ce5582148c6f0c1282335b87df5ed4be4b781399" and content "Some Text" to "/Shares/myChecksumFile.txt" using the WebDAV API
-    Then the HTTP status code should be "204"
-    And as user "Alice" the webdav checksum of "/myChecksumFile.txt" via propfind should match "<checksum>"
-    Examples:
-      | dav_version | checksum                                                                                            |
-      | new         | SHA1:ce5582148c6f0c1282335b87df5ed4be4b781399 MD5:56e57920c3c8c727bfe7a5288cdf61c4 ADLER32:1048035a |
-
```
Comment (grgprarup) [inline tests/acceptance/features/apiMain/checksums.feature:172]: We have a bug demo for issue 38835. Not to remove.

File: tests/acceptance/features/apiProvisioningGroups-v1/deleteGroup.feature [hunk 14]
```diff
@@ -74,19 +74,6 @@ Feature: delete groups
...
-    When the administrator deletes group "<group_id>" using the provisioning API
-    Then the OCS status code should be "100"
-    And the HTTP status code should be "200"
-    And group "<group_id>" should not exist
-    Examples:
-      | group_id         | comment                            |
-      | Mgmt/Sydney      | Slash (special escaping happens)   |
-      | Mgmt//NSW/Sydney | Multiple slash                     |
-      | priv/subadmins/1 | Subadmins mentioned not at the end |
-
```
Comment (grgprarup) [inline tests/acceptance/features/apiProvisioningGroups-v1/deleteGroup.feature:89]: We have a bug demo for issue 31015. Not to remove.

File: tests/acceptance/features/apiProvisioningGroups-v1/deleteGroup.feature [hunk 15]
```diff
@@ -106,14 +93,3 @@ Feature: delete groups
...
-  @issue-31015 @skipOnOcV10 @toImplementOnOCIS
-  Scenario Outline: admin deletes a group that has a forward-slash in the group name
-    Given group "<group_id>" has been created
-    When the administrator deletes group "<group_id>" using the provisioning API
-    Then the OCS status code should be "100"
-    And the HTTP status code should be "200"
-    And group "<group_id>" should not exist
-    Examples:
-      | group_id   | comment             |
-      | var/../etc | using slash-dot-dot |
```
Comment (grgprarup) [inline tests/acceptance/features/apiProvisioningGroups-v1/deleteGroup.feature:119]: We have a bug demo for issue 31015. Not to remove.

File: tests/acceptance/features/apiProvisioningGroups-v1/getUserGroups.feature [hunk 16]
```diff
@@ -91,44 +91,6 @@ Feature: get user groups
...
-    Then the OCS status code should be "100"
-    And the HTTP status code should be "200"
-    And the groups returned by the API should be
-      | brand-new-group      |
-      | 0                    |
-      | Admin & Finance (NP) |
-      | admin:Pokhara@Nepal  |
-      | नेपाली               |
-      | 😅 😆                |
-      | users                |
```
Comment (grgprarup) [inline tests/acceptance/features/apiProvisioningGroups-v1/getUserGroups.feature:120]: We have a bug demo for issue 31015. Not to remove.

File: tests/acceptance/features/apiWebdavPreviews/previewsAutoAdustedSizing.feature [hunk 17]
```diff
@@ -9,17 +9,3 @@ Feature: sizing of previews of files downloaded through the webdav API
...
-    When user "Alice" downloads the preview of "/parent.txt" with width <request_width> and height <request_height> using the WebDAV API
-    Then the HTTP status code should be "200"
-    And the downloaded image should be <return_width> pixels wide and <return_height> pixels high
-    Examples:
-      | request_width | request_height | return_width | return_height |
-      | 1             | 1              | 16           | 16            |
-      | 32            | 32             | 32           | 32            |
-      | 1024          | 1024           | 640          | 480           |
-      | 1             | 1024           | 16           | 16            |
-      | 1024          | 1              | 640          | 480           |
```
Comment (grgprarup) [inline tests/acceptance/features/apiWebdavPreviews/previewsAutoAdustedSizing.feature:25]: Need to verify before removing these tests.

Comment (roshanlc) [reply tests/acceptance/features/apiWebdavPreviews/previewsAutoAdustedSizing.feature]: keeping it for now

File: tests/acceptance/features/apiShareManagementToShares/moveShareInsideAnotherShare.feature [hunk 18]
```diff
@@ -95,13 +95,3 @@ Feature: moving a share inside another share
...
-
-  @skipOnOcV10
-  Scenario: share receiver tries to move a whole share inside a local folder
-    Given user "Brian" has created folder "localFolder"
-    And user "Brian" has uploaded file with content "local text" to "/localFolder/localFile.txt"
-    # On oCIS you cannot move received shares out of the "Shares" folder
-    When user "Brian" moves folder "Shares/folderB" to "localFolder/folderB" using the WebDAV API
-    Then the HTTP status code should be "403"
-    And as "Alice" file "/folderB/fileB.txt" should exist
-    And as "Brian" file "/Shares/folderB/fileB.txt" should exist
```
Comment (grgprarup) [inline tests/acceptance/features/apiShareManagementToShares/moveShareInsideAnotherShare.feature:107]: :+1: This scenario can be removed as we have the same scenarios for the core in Line45.

File: tests/acceptance/features/apiProvisioningGroups-v1/getUserGroups.feature [hunk 19]
```diff
@@ -144,20 +134,3 @@ Feature: get user groups
...
-    And group "group2" has been created
-    And user "Alice" has been added to group "group1"
-    And user "Alice" has been added to group "group2"
-    When user "Alice" gets all the groups of user "Alice" using the provisioning API
-    Then the OCS status code should be "100"
-    And the HTTP status code should be "200"
-    And the groups returned by the API should be
-      | group1 |
-      | group2 |
-      | users  |
```
Comment (grgprarup) [inline tests/acceptance/features/apiProvisioningGroups-v1/getUserGroups.feature:163]: :1:

File: tests/acceptance/features/apiProvisioningGroups-v2/getUserGroups.feature [hunk 20]
```diff
@@ -91,44 +91,6 @@ Feature: get user groups
...
-  @skipOnOcV10
-  Scenario: admin gets groups of an user who is not in any groups on ocis
-    Given user "brand-new-user" has been created with default attributes and without skeleton files
-    And group "unused-group" has been created
-    When the administrator gets all the groups of user "brand-new-user" using the provisioning API
-    Then the OCS status code should be "200"
-    And the HTTP status code should be "200"
-    And the groups returned by the API should be
-      | users |
-
```
Comment (grgprarup) [inline tests/acceptance/features/apiProvisioningGroups-v2/getUserGroups.feature:131]: :+1:

File: tests/acceptance/features/apiProvisioningGroups-v2/getUserGroups.feature [hunk 21]
```diff
@@ -144,20 +106,3 @@ Feature: get user groups
...
-    And group "group2" has been created
-    And user "Alice" has been added to group "group1"
-    And user "Alice" has been added to group "group2"
-    When user "Alice" gets all the groups of user "Alice" using the provisioning API
-    Then the OCS status code should be "200"
-    And the HTTP status code should be "200"
-    And the groups returned by the API should be
-      | group1 |
-      | group2 |
-      | users  |
```
Comment (grgprarup) [inline tests/acceptance/features/apiProvisioningGroups-v2/getUserGroups.feature:163]: :+1:

File: tests/acceptance/features/apiAuth/cors.feature (hunk 2 above)
Comment (grgprarup) [reply tests/acceptance/features/apiAuth/cors.feature]: @phil-davis we will remove the `notToImplementOnOCIS` and `skipOnOcis` tags in the next PR.

File: tests/acceptance/features/apiProvisioningGroups-v1/getUserGroups.feature (hunk 16 above)
Comment (grgprarup) [reply tests/acceptance/features/apiProvisioningGroups-v1/getUserGroups.feature]: Sorry, my mistake this scenario can be removed as we have this scenario for core in L11 and this scenario is only for ocis.

**Review by saw-jan — COMMENTED** [review]
LGTM

File: tests/acceptance/features/apiProvisioning-v1/resetUserPassword.feature [hunk 22]
```diff
@@ -93,12 +93,6 @@ Feature: reset user password
       | ⌚️ 📱 📲 💻   | objects |
       | 🚴🏿‍♀️ 🚴‍♂️ | cycling |
 
-  @skipOnOcV10 @issue-37992
```
Comment (phil-davis) [inline tests/acceptance/features/apiProvisioning-v1/resetUserPassword.feature:96]: Note: this is OK to delete. #37992 is closed.

File: tests/acceptance/features/apiProvisioning-v2/resetUserPassword.feature [hunk 23]
```diff
@@ -93,12 +93,6 @@ Feature: reset user password
       | ⌚️ 📱 📲 💻   | objects |
       | 🚴🏿‍♀️ 🚴‍♂️ | cycling |
 
-  @skipOnOcV10 @issue-37992
```
Comment (phil-davis) [inline tests/acceptance/features/apiProvisioning-v2/resetUserPassword.feature:96]: Note: this is OK to delete. https://github.com/owncloud/core/issues/37992 is closed.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**